FinancialTracker/Cutsom UI/TransferTableViewCell.swift

```diff
@@ -7,7 +7,21 @@
 
 import UIKit
 
+protocol TransferTableViewCellDelegate: AnyObject {
+    func didTapTransferStateButton(with title: String, section: Int, row: Int)
+}
+
 class TransferTableViewCell: UITableViewCell {
     @IBOutlet var transferTitleLabel: UILabel!
     @IBOutlet var transferDate: UILabel!
+    @IBOutlet var transferStateButton: UIButton!
+    
+    weak var delegate: TransferTableViewCellDelegate?
+    
+    var section: Int?
```

Comment:
I need to know if the button is selected, not the row. Also in many forums it was recommend with delegate

Reply:
Your only clickable element is the button in the cell. No other. So, you'll be fine. Also, section and row should not be kept in the cell itself because it is pointless most of all. In general, we want to use what we have available already, we do not want to duplicate available functionalities